setup.py

```diff
@@ -545,18 +598,24 @@ def run(self):
             "build/pip_data_bin_init.py.in",
             os.path.join(bin_dir, "__init__.py"),
         )
+        # Share the cmake-out location with _BaseExtension.
+        self.cmake_cache_dir = cmake_cache_dir
 
         # Finally, run the underlying subcommands like build_py, build_ext.
         build.run(self)
 
 
 def get_ext_modules() -> List[Extension]:
     """Returns the set of extension modules to build."""
-
     ext_modules = []
     if ShouldBuild.flatc():
         ext_modules.append(
-            BuiltFile("third-party/flatbuffers/flatc", "executorch/data/bin/")
+            BuiltFile(
+                "third-party/flatbuffers/%BUILD_TYPE%/",
+                "flatc",
+                "executorch/data/bin/",
```

Comment:
Now that there are more than two params, please use kwargs for all params. Same for other BuiltFile uses below.

```suggestion
                src_path="third-party/flatbuffers/%BUILD_TYPE%/",
                src_name="flatc",
                dst="executorch/data/bin/",
```